api_tests/src/remote_home_remote.spec.ts

```diff
@@ -0,0 +1,952 @@
+/*
+    remote-home-remote Lemmy Instance behavior testing
+    Tests are intended to emphasize:
+       1. end-user, non-admin, accounts
+       2. lemmy-beta instance being the "home instance" of a community
+       3. lemmy-alpha and lemmy-gamma being "remote instances" of a community.
+       4. lemmy-beta Rust code paths that replicate data between lemmy-beta & lemmy-gamma instances
+*/
+jest.setTimeout(90 * 1000);
```

Comment:
This file looks helpful. Please change it so that individual tests can be run, not only the whole file at once. You can do that by changing line 12 in package.json like this:
```
    "api-test": "jest -i remote_home_remote.spec.ts -t 'non-moderator user of alpha creates post, moderator of alpha community-features, gamma reads it'"
```

It currently fails because `alphaCommunityRemote` gets initialized in another test case, but it should be done in `beforeAll()`. Once thats working I can debug and see where the problem is.